packages/foundation-models/README.md

```diff
@@ -60,26 +60,40 @@ const chatClient = new AzureOpenAiChatClient({
 });
 ```
 
-### Azure OpenAI Client
+### Chat Client
 
-The Azure OpenAI client can then be used to send chat completion or embedding requests to models deployed in the SAP generative AI hub.
+Use the `AzureOpenAiChatClient` to send chat completion requests to an OpenAI model deployed in SAP generative AI hub.
 
-#### Chat Client
+The client sends request with Azure OpenAI API version `2024-06-01`.
 
-Use the `AzureOpenAiChatClient` to send chat completion requests to an OpenAI model deployed in SAP generative AI hub.
+Set request configuration in the `requestConfig` parameter. The following example shows a call with messages and optional request configuration.
 
 ```ts
 import { AzureOpenAiChatClient } from '@sap-ai-sdk/foundation-models';
 
 const chatClient = new AzureOpenAiChatClient('gpt-4o');
-const response = await chatClient.run({
-  messages: [
-    {
-      role: 'user',
-      content: 'Where is the deepest place on earth located'
+const response = await chatClient.run(
+  {
+    messages: [
+      {
+        role: 'user',
+        content: 'Where is the deepest place on earth located'
+      }
+    ]
+  },
+  {
+    headers: {
+      'content-type': 'application/json',
```

Comment:
[req] Aren't these headers already sent by default anyway? Can we just give an example of a custom dummy header? `content-type`  and `ai-client-type` are passed with these values, and `ai-resource-group` is meant to be set at the time of initialization. Setting it here is technically incorrect.

Reply:
True. Thanks for pointing it out.